packages/block-editor/src/components/off-canvas-editor/index.js

```diff
@@ -104,9 +108,9 @@ function __ExperimentalOffCanvasEditor(
 		setExpandedState,
 	} );
 	const selectEditorBlock = useCallback(
-		( event, clientId ) => {
-			updateBlockSelection( event, clientId );
-			setSelectedTreeId( clientId );
+		( event, _clientId ) => {
```

Comment:
These changes to `clientID` were to appease the linter which was telling me it was shadowing an existing var.